moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -165,7 +192,35 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
   }
   ROS_DEBUG_NAMED("chomp_planner", "Optimization took %f sec to create", (ros::WallTime::now() - create_time).toSec());
 
-  optimizer.optimize();
+  int optimization_result = optimizer.optimize();
+
+  /// replan with updated parameters if no solution is found
```

Comment:
only two slashes //, three is reserved for doxygen comments in the header file for public functions

Reply:
done